source/common/tcp_proxy/upstream.cc

```diff
@@ -195,8 +195,14 @@ HttpConnPool::HttpConnPool(Upstream::ThreadLocalCluster& thread_local_cluster,
                            Tcp::ConnectionPool::UpstreamCallbacks& upstream_callbacks,
                            Http::CodecType type)
     : config_(config), type_(type), upstream_callbacks_(upstream_callbacks) {
-  conn_pool_data_ = thread_local_cluster.httpConnPool(Upstream::ResourcePriority::Default,
-                                                      absl::nullopt, context);
+  absl::optional<Http::Protocol> protocol;
+  if (type_ == Http::CodecClient::Type::HTTP3) {
```

Comment:
nit: Http::CodecType::HTTP3 etc?

I unified types recently and left old aliases to not break downstream but I'd prefer Envoy be consistent

Reply:
Done